content/CORE/Network/Interfaces/_index.md

```diff
@@ -2,4 +2,108 @@
 title: "Interfaces"
 geekdocCollapseSection: true
 weight: 30
----
\ No newline at end of file
+aliases:
+  - /core/network/interfaces/laggcreate/
+  - /core/network/interfaces/bridgecreate/
+  - /core/network/interfaces/vlancreate/
+---
+
+The TrueNAS CORE **Interfaces** section displays interface names and IP addresses, as well as their types and link states. 
+
+## Adding, Editing, and Deleting Interfaces
+
+To add a new network interface, click **ADD**.
+
+To edit or delete an interface, click the <i class="material-icons" aria-hidden="true" title="Options">more_vert</i> in that interface's row, then click <i class="material-icons" aria-hidden="true" title="edit">edit</i> **EDIT** or <i class="material-icons" aria-hidden="true" title="delete">delete</i> **DELETE**.
+
+You can click <i class="material-icons" aria-hidden="true" title="edit">edit</i> **EDIT** on a phycial interface to [set a static IP address for the TrueNAS UI]({{< relref "EditingPhysicalInterface.md" >}})
+
+You can also [edit your TrueNAS sytem's physical NICs]({{< relref "SettingStaticIP.md" >}}) by clicking <i class="material-icons" aria-hidden="true" title="delete">delete</i> **RESET CONFIGURATION**.
+
+{{< hint warning >}} 
+Be careful when configuring the network interface that controls the TrueNAS® web interface or you may lose web connectivity. 
+{{< /hint >}}
+
+## Creating LAGG, Bridge, and VLAN Interfaces
+
+{{< expand "Why should I use different interface types?" "v" >}}
+{{< include file="/_includes/NetworkInterfaceTypes.md" type="page" >}}
+{{< /expand >}}
+
+{{< tabs "Interfaces" >}}
+{{< tab "LAGG" >}}
+A [Link Aggregation (LAGG)](https://tools.ietf.org/html/rfc7424) is a general method of combining (aggregating) multiple network connections in parallel or a series to provide additional bandwidth or redundancy for critical networking situations. TrueNAS uses [lagg(4)](https://www.freebsd.org/cgi/man.cgi?lagg(4)) to manage LAGGs.
+
+To set up a LAGG interface, go to **Network > Interface > Add**.
+
+![NetworkInterfacesAddLAGG](/images/CORE/12.0/NetworkInterfacesAddLAGG.png "Adding a lagg interface")
+
+Set the *Type* to *Link Aggregation*.
+
+Enter a name for the interface. The name must use the format *laggX*, where *X* is a number representing a non-parent interface.
+It is also recommended to add any notes or reminders about this particular LAGG in the *Description*.
+
+Under **LAGG Settings**, set the *Lagg Protocol* to configure the interface ports to match your networking needs:
+
+{{< tabs "Lagg Protocol Options" >}}
+{{< tab "LACP" >}}
```

Comment:
We can't nest tab boxes inside tab boxes 😢 . We'll need to rework the presentation of this info a different way.

Reply:
Don't know how I missed that. Fixed!